src/docker.js

```diff
@@ -105,6 +106,9 @@ docker.buildImage = function(project, tarPath, imageId, buildId, buildLogger, do
         }
 
         buildLogger.info('[%s] %s', tag, result.stream || result.status);
+        if (result.stream && result.stream.indexOf('Successfully built ') == 0) {
+          realBuildId = result.stream.split('Successfully built ')[1].replace('\n', '');
```

Comment:
Couldn't find a cleaner way around it. To avoid potential issue, did set the `realBuildId` as `buildID` as initial value.